tests/unit/net/fetch.rs

```diff
@@ -203,6 +204,7 @@ fn test_cors_preflight_fetch() {
         if request.method == Method::Options && state.clone().fetch_add(1, Ordering::SeqCst) == 0 {
             assert!(request.headers.has::<AccessControlRequestMethod>());
             assert!(request.headers.has::<AccessControlRequestHeaders>());
+            assert!(request.headers.has::<HyperReferer>());
```

Comment:
You'll want the [get method](http://doc.servo.org/hyper/header/struct.Headers.html#method.get) and then look at the [internal value](http://doc.servo.org/hyper/header/struct.Referer.html).